lib/octicons_react/src/createIconComponent.js

```diff
@@ -12,24 +12,31 @@ export function createIconComponent(name, defaultClassName, getSVGData) {
 
   function Icon({
     'aria-label': ariaLabel,
+    'aria-labelledby': arialabelledby,
     tabIndex,
     className = defaultClassName,
     fill = 'currentColor',
     size = 16,
+    ref,
```

Comment:
I think `ref` should be covered in the latest updates from `main` with `React.forwardRef`: https://github.com/primer/octicons/blob/main/lib/octicons_react/src/createIconComponent.js#L23 once the branch is updated with latest 👀